src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionPool.cs

```diff
@@ -367,10 +378,54 @@ public byte[] Http2AltSvcOriginUri
             }
         }
 
-        public bool EnableMultipleHttp2Connections => _poolManager.Settings.EnableMultipleHttp2Connections;
+        private bool EnableMultipleHttp2Connections => _poolManager.Settings.EnableMultipleHttp2Connections;
```

Comment:
Why do we even keep it? It's used at only one place.

Reply:
Yeah, I will remove this.